monai/transforms/intensity/array.py

```diff
@@ -800,48 +956,83 @@ def __call__(self, img: NdarrayOrTensor) -> NdarrayOrTensor:
 
 class AdjustContrast(Transform):
     """
-    Changes image intensity by gamma. Each pixel/voxel intensity is updated as::
+    Changes image intensity with gamma transform. Each pixel/voxel intensity is updated as::
 
         x = ((x - min) / intensity_range) ^ gamma * intensity_range + min
 
     Args:
         gamma: gamma value to adjust the contrast as function.
+        invert_image: multiplies all intensity values with -1 before gamma transform and again after gamma transform
```

Comment:
```suggestion
        invert_image: whether to invert the image before applying gamma augmentation. If True, multiply all intensity values with -1 before the gamma transform and again after the gamma transform.
```

Reply:
Maybe we can add a reference link here?

Reply:
Do you mean a reference to the corresponding nnU-Net function? 
https://github.com/MIC-DKFZ/batchgenerators/blob/7fb802b28b045b21346b197735d64f12fbb070aa/batchgenerators/augmentations/color_augmentations.py#L107

Reply:
Yes, it's used there. Ok, I will include both links.